### PR TITLE
[WIP] trying to use DefaultValue instead of Value

### PR DESCRIPTION
**Pull request by et1975**

### CODE REVIEW

**Comment by et1975** [comment]
Hmm, CI is broken for unrelated reasons, @forki could you give it a try?

**Comment by alfonsogarciacaro** [comment]
I rebuilt the PR and at least in Appveyor is working, but this is still the old fable-elmish version, right?

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "fable-core": "^0.7.28",
     "fable-elmish": "^0.8.1",
     "fable-elmish-debugger": "^0.1.2",
-    "fable-elmish-react": "0.8.2",
+    "fable-elmish-react": "^0.8.2",
```
Comment (et1975) [inline package.json]: Reverted this, but it doesn't show up in the diff...

**Comment by alfonsogarciacaro** [comment]
Thanks! Let's merge this :+1:

**Comment by et1975** [comment]
Yeah... looking at the file it's still referencing the old version :(

**Pull request merged**